tests/conftest.py

```diff
@@ -82,3 +82,13 @@ def connect_to_slave(pytestconfig, read_config):
 
     yield servo, net
     net.disconnect_from_slave(servo)
+
+
+@pytest.fixture()
+def virtual_drive():
+    test_ip = "localhost"
```

Comment:
Maybe you could use the IP address 127.0.0.1 so that the variable name would make more sense.